Analytical_oM/Results/IObjectIdResult.cs

```diff
@@ -26,15 +26,18 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.ComponentModel;
 
 namespace BH.oM.Analytical.Results
 {
+    [Description("Interface for Results corresponding to a particular object that is to be identified and correlated to the object by its id.")]
```

Comment:
```suggestion
    [Description("Interface for Results belonging to a particular object. The owning object is identified and correlated to the Results by its id.")]
```
This is a tricky one